### PR TITLE
The magic of parallel-execution (pylint)

### PR DESCRIPTION
**Pull request by PixelyIon**
http://pylint.pycqa.org/en/latest/user_guide/run.html#parallel-execution

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,9 +5,9 @@ python:
 script:
   - # @TODO run tests here
   - pip install .
-  - "nosetests && pylint pommerman"
-  - "nosetests && pylint mananger"
-  - "nosetests && pylint examples"
+  - "nosetests && pylint -j 0 pommerman"
+  - "nosetests && pylint -j 0 mananger"
+  - "nosetests && pylint -j 0 examples"
```
Comment (weseldridge) [inline .travis.yml]: So this is just running on 1 thread. Will this really make this faster?

Comment (PixelyIon) [reply .travis.yml]: The `-j 0` parameter auto-detects the amount of cores and it to that.

Comment (cinjon) [reply .travis.yml]: Is that ... desirable? It means that suddenly all of your cores are going to this without being notified. OTOH, it is a really lightweight task.

**Comment by cinjon** [comment]
Letting this through. It sounds nice to have this process go quicker; I'll wait and see if it causes problems for anyone.

**Pull request merged**